app/views/admin/schedules/show.html.haml

```diff
@@ -1,48 +1,166 @@
-.schedule-content
-  %h2
-    = "Schedule for #{@conference.title}"
-  #unscheduled.unscheduled
-    .unscheduled-header
-      Unscheduled Events
-  #schedule.schedule
-    .schedule-dates-header
-      %ul#date-tabs
-        - @dates.each do |date|
-          %li.date-selector
-            = link_to "#{date}", "#", id: "#{date}-selector", onclick: "Schedule.changeDay('#{date}')"
-    .schedule-rooms-container
-      = render 'day_tab'
+.row
+  .col-md-12
+    .page-header
+      %h1 Schedule
+      %p.text-muted
+        Create the schedules for the conference
+
+.row
+  .col-md-2
+    .h4
+      Unschedelud events
+    .unscheduled-events
+  .col-md-10
+    %ul.nav.nav-tabs
+      - @dates.each do |date|
+        %li{ class: "#{ (@dates.first == date) ? 'active' : '' }"}
+          %a{ href: "##{date}" }
+            = date
+    .tab-content
+      - @dates.each do |date|
+        .tab-pane{ class: "#{ (@dates.first == date) ? 'active' : '' }", id: "#{date}" }
+          = render partial: 'day_tab', locals: { date: date }
+
 
 :javascript
+
```

Comment:
You should move the JS in a dedicated .js file. Inline JS is bad style and hard to read / review!